### PR TITLE
Fix Live Merge Page

### PR DESCRIPTION
**Pull request by mureinik**
Changes proposed in this pull request:
- Fix markdown issue
- Fix feature page header
- Update the correct status

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @mureinik

This pull request needs review by: @aglitke @alhino

### CODE REVIEW

File: source/develop/release-management/features/storage/live-merge.html.md [hunk 1]
```diff
@@ -19,30 +20,28 @@ Live merge makes it possible to delete virtual machine disk snapshots that are n
...
+A live merge is initiated when a user clicks the 'Remove' command associated with a disk snapshot. If the virtual machine is down, the existing col merge flow is followed. If the virtual machine is up, a live merge is attempted. The engine calls the VDSM merge() verb on the host on which the virtual machine is running. First, VDSM associates a new `blockJob` record with the virtual machine. This record is initially populated with placeholder values so that it can serve as a record that a job is just beginning. Next, VDSM calls the libvirt `virDomainBlockCommit` API to start the merge. This libvirt call is asynchronous and its completion is signaled by an event (ignored by VDSM) and the job being no longer reported by the `virDomainBlockJobInfo` API. Finally, VDSM returns success to engine. Some considerations regarding race conditions with reporting block job information via the virtual machine Stats mechanism will appear later in this page.
```
Comment (rollandf) [inline source/develop/release-management/features/storage/live-merge.html.md]: the existing col merge flow - > the existing cold merge flow

Comment (mureinik) [inline source/develop/release-management/features/storage/live-merge.html.md]: Good catch, thanks. Will post an updated series when I get back home and have more reliable wifi.

**Comment by mureinik** [comment]
@rollandf @alhino @aglitke - ping? Do you have any other suggestions for this PR, or can we move forward and merge it?

**Comment by rollandf** [comment]
👍

**Pull request merged**